music/src/spice/spice.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ *
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as tf from '@tensorflow/tfjs';
+import { resampleAndMakeMono } from '../core/audio_utils';
+import { computePower } from './loudness_utils';
+import { AudioFeatures } from '../ddsp/interfaces';
+import { getPitches } from './pitch_utils';
+
+// SPICE SPECIFIC CONSTANTS
+const SPICE_MODEL_URL = 'https://tfhub.dev/google/tfjs-model/spice/2/default/1';
```

Comment:
I like that you made a spice model class. Wouldn't it make sense to have these be default args of the Spice model, rather than in this separate file? and to have the functions be methods of the model? That way it has the same api as the other models (takes in a url in the constructor, but with a default to this url).

Reply:
I think it makes more sense to have this here since this constant won't be modified (and shouldn't be). I think to try to make this class be consistent with the other classes in the Magenta library is a bad idea in this case because the url parameter will be optional and might potentially mislead one into thinking that they can use their own model somehow. 

I'm in favor to keep it this way unless you feel strongly about this.

Reply:
on second thought, users might want to avoid using tfhub altogether. i've made the changes as you suggested!